LayoutTests/fast/images/text-recognition/image-overlay-with-attachment-element.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html><!-- webkit-test-runner [ AttachmentElementEnabled=true AttachmentWideLayoutEnabled=true ] -->
+<html>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<head>
+<script src="../../../resources/js-test.js"></script>
+<style>
+img {
+    width: 100px;
+    height: 100px;
+    box-sizing: border-box;
+    border: 1px solid black;
+    display: block;
+}
+</style>
+<script>
+addEventListener("load", () => {
+    let image = document.getElementById("inside-link");
+    internals.ensureUserAgentShadowRoot(image);
+    const attachment = document.createElement('attachment');
+    attachment.setAttribute('type', 'public.jpeg');
+    attachment.setAttribute('title', 'green-400x400.png');
+    attachment.setAttribute('style', 'display:none !important');
+    internals.shadowRoot(image).appendChild(attachment)
+    internals.installImageOverlay(image, [
+        {
+            topLeft : new DOMPointReadOnly(0, 0.3),
+            topRight : new DOMPointReadOnly(1, 0.3),
+            bottomRight : new DOMPointReadOnly(1, 0.7),
+            bottomLeft : new DOMPointReadOnly(0, 0.7),
+            children: [{
+                text : "Hello",
+                topLeft : new DOMPointReadOnly(0, 0.3),
+                topRight : new DOMPointReadOnly(1, 0.3),
+                bottomRight : new DOMPointReadOnly(1, 0.7),
+                bottomLeft : new DOMPointReadOnly(0, 0.7),
+            }],
+        }
+    ]);
+    const rect = image.getBoundingClientRect();
+    eventSender.mouseMoveTo(rect.left + rect.width / 2, rect.top + rect.height / 2);
+    const overlay = internals.shadowRoot(image).getElementById("image-overlay");
+    transform = getComputedStyle(overlay.childNodes[0].childNodes[0]).transform;
+    noTransform = "none";
+    shouldNotBe("transform", "noTransform");
```

Comment:
Nit - this seems a bit clunky to me. Maybe something like this?
```suggestion
    shouldNotBeEqualToString("transform", "none");
```

Reply:
I thought about this but then looked in `js-test.js` and shouldNotBeEqualToString isn't defined in it?

Reply:
> I thought about this but then looked in `js-test.js` and shouldNotBeEqualToString isn't defined in it?

Oh, maybe it's `js-test-pre.js` only. In that case we can either add it to `js-test.js` or simply use `shouldNotBe("transform", "'none'");`